src/commands/RequestDiscordAccessCommand.ts

```diff
@@ -0,0 +1,84 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { CommandInteraction, GuildMember } from 'discord.js';
+import { ExecuteFunction } from '../interfaces/Command';
+import * as students from '../../assets/Listado.json';
+
+function emailIsValid(email: string) {
+    const re =
+        /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    return re.test(String(email).toLowerCase());
```

Comment:
`.toString()` creo que seria mas correcto que usar `String`.
```suggestion
    return re.test(email.toString().toLowerCase());
```

De todas maneras, el tipo no dice que email ya es un string?